### ISSUE TITLE
support  the couchbase search driver ?

### ISSUE THREAD

**Issue by joeblew99** [dataux/dataux#46]
there is a driver here that conforms to the golang SQL interface

https://github.com/couchbase/go_n1ql
works very well.

**Comment by araddon** [comment]
yes, i have seen that before, ill add it to (newly created)  https://github.com/dataux/dataux/projects/1 roadmap for sources, thx for input.
